source/common/quic/envoy_quic_utils.cc

```diff
@@ -223,5 +224,33 @@ int deduceSignatureAlgorithmFromPublicKey(const EVP_PKEY* public_key, std::strin
   return sign_alg;
 }
 
+void configQuicInitialFlowControlWindow(const envoy::config::core::v3::QuicProtocolOptions& config,
+                                        quic::QuicConfig& quic_config) {
+  // Use HTTP2 minimum value instead of default ones because the defaults are way too large.
```

Comment:
What are good defaults to use for QUIC?  Are the QUIC maximums better defaults than the proposed 64kb?